src/metabase/api/card.clj

```diff
@@ -166,6 +166,7 @@
                  (hydrate :creator
                           :bookmarked
                           :dashboard_count
+                          :filter_values_usage_count
```

Comment:
```suggestion
                          :parameter_card_usage_count
```
does this make more sense? since this comes from parameter_card table.

Reply:
in this case, I think frontend calls it `Widget dropdown` or something similar.

Reply:
and `filter` is not technically correct once we support `custom values` on data-apps. in that case it's more like `selection-values`

Reply:
Renamed to `dropdown_widget_values_usage_count` etc.